utils/scripts/dbaas_utils.py

```diff
@@ -224,25 +224,28 @@ def retrieve_instances(isv, namespace):
         create_secret_cli(isv, "true")
         import_provider_account_cli(isv, namespace)
         provider_account = BuiltIn().get_variable_value(r"\${provaccname}")
-    while counter < 5:
+    while counter < 3:
```

Comment:
Why is it 3 instead of 5 now?

Reply:
This is on number of retries for expected status of a resource. I cut it short so we would be waiting for a minute, earlier it was more than 2 mins.